.travis.yml

```diff
@@ -11,6 +11,7 @@ matrix:
                   packages:
                       - g++-6
                       - libcppunit-dev
+                      - gettext
```

Comment:
@neheb  Please rework your commit comment - or at least the description of the PR - to explain **why** you add gettext. Your current commit comment only paraphrase the code without giving a reason. Thanks!